StabilityMatrix.Core/Models/Packages/SimpleSDXL.cs

```diff
@@ -0,0 +1,150 @@
+﻿using StabilityMatrix.Core.Attributes;
+using StabilityMatrix.Core.Helper;
+using StabilityMatrix.Core.Helper.Cache;
+using StabilityMatrix.Core.Helper.HardwareInfo;
+using StabilityMatrix.Core.Models.FileInterfaces;
+using StabilityMatrix.Core.Models.Progress;
+using StabilityMatrix.Core.Processes;
+using StabilityMatrix.Core.Python;
+using StabilityMatrix.Core.Services;
+
+namespace StabilityMatrix.Core.Models.Packages;
+
+[Singleton(typeof(BasePackage))]
+public class SimpleSDXL(
+    IGithubApiCache githubApi,
+    ISettingsManager settingsManager,
+    IDownloadService downloadService,
+    IPrerequisiteHelper prerequisiteHelper
+) : Fooocus(githubApi, settingsManager, downloadService, prerequisiteHelper)
+{
+    public override string Name => "SimpleSDXL";
+    public override string DisplayName { get; set; } = "SimpleSDXL";
+    public override string Author => "metercai";
+    public override string Blurb =>
+        "Enhanced version of Fooocus for SDXL, more suitable for Chinese and Cloud. Supports Flux.";
+    public override string LicenseUrl => "https://github.com/metercai/SimpleSDXL/blob/SimpleSDXL/LICENSE";
+    public override Uri PreviewImageUri =>
+        new("https://github.com/user-attachments/assets/98715a4d-9f4a-4846-ae62-eb8d69793d31");
+    public override PackageDifficulty InstallerSortOrder => PackageDifficulty.Expert;
+    public override IEnumerable<SharedFolderMethod> AvailableSharedFolderMethods =>
+        [SharedFolderMethod.Symlink, SharedFolderMethod.None];
+    public override SharedFolderMethod RecommendedSharedFolderMethod => SharedFolderMethod.Symlink;
+    public override string MainBranch => "SimpleSDXL";
+
+    public override List<LaunchOptionDefinition> LaunchOptions =>
+        [
+            new()
+            {
+                Name = "Preset",
+                Type = LaunchOptionType.Bool,
+                Options =
+                {
+                    "--preset anime",
+                    "--preset realistic",
+                    "--preset Flux",
+                    "--preset Kolors",
+                    "--preset pony_v6"
+                },
+            },
+            new()
+            {
+                Name = "Language",
+                Type = LaunchOptionType.String,
+                Description = "Change the language of the UI",
+                Options = { "--language" },
+                DefaultValue = "en"
```

Comment:
I admit these could be explained a bit better, but I believe you want `InitialValue` here instead of `DefaultValue`. The default value is used for the placeholder text for when the argument isn't passed at all. The initial value will actually send the argument along with the command to run the package.

Also I noticed that `--disable-preset-download` worked here (I don't want juggernaut dangit 😛), are there any other ones you could copy from the base Fooocus?

Reply:
That's interesting. With `DefaultValue` the UI actually switched to English on launch. Without it, it was always Chinese. But Ill test that tomorrow. Ill check the other flags as well tomorrow (e.g. disable-preset-download) 😄

Reply:
Would it make sense to have both Default and Initial so the user knows what the initial one is?